doc/2/api/controllers/document/search/index.md

```diff
@@ -31,7 +31,7 @@ You can restrict the scroll session maximum duration under the `services.storage
 ### HTTP
 
 ```http
-URL: http://kuzzle:7512/<index>/<collection>/_search[?from=<int>][&size=<int>][&scroll=<time to live>]
+URL: http://kuzzle:7512/<index>/<collection>/_search[?from=<int>][&size=<int>][&scroll=<time to live>][&verb=string]
```

Comment:
I don't see the point of passing the `verb` argument in the query string for Http since this information is already present in the native verb used for the request (GET or POST`

Reply:
When you use the SDK and pass the verb option. the sdk will construct the query string like this. That is why I wrote it this way in the doc.
I let it because it is still an option, even if it is not really necessary...

Reply:
The SDK should not pass the `verb` in the query string but it should change the HTTP verb used for the query.  
Having a `verb` option is useless in HTTP since we already have the verb (also cache system as in webworkers don't care about a `verb` argument in the query string for the cache policiy, they only rely on the http verb)